release_notes.md

```diff
@@ -0,0 +1,6 @@
+# Fix Details
+
+- Removed `Scenario` module and SVT from the solution pack
+- Demo alert is now created with type `Suspicious Email` instead of `Phishing`
+- The response playbook `Phishing Email Triage Based On ML Classifier` is modified to execute on `Suspicious Email` alert type
```

Comment:
Correction: The response playbook `Phishing Email Triage Based On ML Classifier` will now execute on Alerts of type`Suspicious Email`

Reply:
Made suggested changes in release notes